en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,474 @@
+
+The following diagram summarizes the use case:
+
+![Use case summary](../assets/img/get-started/use-case-summary.png){.cInlineImage-full}
+
+Let's get started!
+
+## Prerequisites
+
+Before you try out this guide, complete the following:
+
+1. Create a GitHub repository to save the REST API implementation. For this guide, you can fork [https://github.com/Asitha/choreo-reading-list-application-example](https://github.com/Asitha/choreo-reading-list-application-example).
+
+2. If you are logging in to Choreo Console for the first time, create an organization as follows:
+
+    1. In [https://wso2.com/choreo/](https://wso2.com/choreo/), click **Try Choreo Now**.
+
+    2. Sign in to Choreo using your Google, GitHub, or Microsoft account.
+
+    3. Enter a unique organization name. For example, `Stark Industries`.
+
+        ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
+
+    4. Access and read the Privacy Policy and Terms of Use via the provided links, and select the **I agree with the Privacy Policy and Terms of Use** checkbox if you agree with them.
+
+    5. Click **Create** to add the new organization.
+
+    6. When the **Welcome to Choreo!** message appears, click either **Decline** or **Confirm** as you prefer depending on whether you want Choreo to send you emails with product updates, resources, and event invites.
+
+    You will be viewing the Home page of the Choreo Console.
+
+3. If required, create a new project as follows:
+
+    !!! info
+        If you do not create a new project, you can create the REST API for the default project.
+    
+    1. On the Home page of the Choreo Console, click **+ Create Project**.
+
+    2. Enter a unique name and a description for the project.
+
+    3. Click **Create**.
+
+## Step 1: Create and publish a REST API
+
+In this step, you are playing the role of John, the API developer. You will create and publish the REST API that the application developed by Sam needs to consume.
+
+### Step 1.1: Create the REST API
+
+Let's create your first REST API.
+
+1. On the Home page, click on the required project.
+
+2. If your project already has components, click **Create** and then click **Create** again in the REST API card. If not, click **Create** in the  REST API card.
+
+3. Enter a unique name and a description for the API. For example, you can enter the name and the description given below:
+
+    | **Field**       | **Value**               |
+    |-----------------|-------------------------|
+    | **Name**        | `readingList`           |
+    | **Description** | `Manages reading lists` |
+
+4. In the **Access Mode** field, leave the default selection unchanged so that users outside your organization can access your API.
+
+5. Click **Next**.
```

Comment:
We can merge step 5 and 6  isn't it?
Also, got to explain a bit about why `Authorize with Github`?? wdyt? Maybe in the info box.

Reply:
As a novice user, I felt we can answer the `why` question on this a bit more.

Reply:
Updated the description to answer the `why` question.

Reply:
> We can merge step 5 and 6 isn't it? Also, got to explain a bit about why `Authorize with Github`?? wdyt? Maybe in the info box.

Do you mean like "Click **Next** and then click **Authorize with Github**."?

Now step 6 is changed to "To allow Choreo to connect to your GitHub account, click **Authorize with GitHub** to answer "why?". Therefore, kept it as a separate step.